<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_m_member_point_recharge_Dialog</class>
 <widget class="QDialog" name="w_m_member_point_recharge_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="outer_stylesheet" stdset="0">
      <string>frame_detail_show</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>积分情况</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_vch_memberno">
        <property name="text">
         <string>会员编号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_vch_memberno">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>积分类型</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_ch_type">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>原积分值</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="lds_doublespinbox_lineedit" name="lineEdit_num_point_org">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>操作积分</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="lds_doublespinbox" name="lineEdit_num_point"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_memo">
        <property name="text">
         <string>备注</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="lds_ComboBox" name="comboBox_vch_memo">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="duplicatesEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>操作员</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_vch_operid">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>操作时间</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_dt_operdate">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="lds_toolbar" name="widget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar</class>
   <extends>QFrame</extends>
   <header>lds_toolbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>lds_ComboBox</class>
   <extends>QComboBox</extends>
   <header>lds_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>lds_doublespinbox_lineedit</class>
   <extends>QLineEdit</extends>
   <header>lds_doublespinbox_lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>lds_doublespinbox</class>
   <extends>QDoubleSpinBox</extends>
   <header>lds_doublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_ch_type</tabstop>
  <tabstop>lineEdit_num_point_org</tabstop>
  <tabstop>comboBox_vch_memo</tabstop>
  <tabstop>lineEdit_vch_operid</tabstop>
  <tabstop>lineEdit_dt_operdate</tabstop>
  <tabstop>lineEdit_vch_memberno</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
